meet_balls_app/templates/meet_balls_app/profile.html

```diff
@@ -0,0 +1,85 @@
+{% extends 'base.html' %} {% block content %}
+<h1>Welcome, {{ player.user.username }} !</h1>
+<h2>Profile Information</h2>
+<p>Date of Birth: {{ player.birth_date }}</p>
+<p>Favorite Ball Game: {{ player.favorite_ball_game }}</p>
+<a class="btn btn-primary mb-2" href="{% url 'edit profile' player.user.id %}"
+  >Edit Profile</a
+>
+<h2>Player Ratings</h2>
+<table>
+  <thead>
+    <tr>
+      <th>Ball Game</th>
+      <th>Rating</th>
+    </tr>
+  </thead>
+  <tbody>
+    {% for rating in ratings %}
+    <tr>
+      <td>{{ rating.ball_game }}</td>
+      <td>{{ rating.rating }}</td>
+    </tr>
+    {% endfor %}
+  </tbody>
+</table>
+
+<h2>Game History</h2>
+<div>
+  <canvas id="gameHistoryChart"></canvas>
+</div>
+
+<script src="https://cdn.jsdelivr.net/npm/chart.js"></script>
```

Comment:
can you please add a script block in the main html file and then import this section into that

it's best to have the script as part of the HEAD and not inside the body

Reply:
didn't fully understand, can you please elaborate ?

Reply:
I saw you move it to base I don't think that is right because we don't need to js always in all the packages

what you can do is something like

```
<head>
{% block scripts %} {% endblock %}
</head>
```
in the base and the extend scripts block in the profile htlm with that line :)

Reply:
done